clightning/options.go

```diff
@@ -10,7 +10,7 @@ const (
 	liquidRpcUserOption             = "peerswap-elementsd-rpcuser"
 	liquidRpcPasswordOption         = "peerswap-elementsd-rpcpassword"
 	liquidRpcPasswordFilepathOption = "peerswap-elementsd-rpcpasswordfile"
-	liquidDisabledOption            = "peerswap-elementsd-disabled"
+	liquidEnabledOption             = "peerswap-elementsd-enabled"
```

Comment:
Same background.
How about the same name?
```suggestion
	liquidSwapsOption             = "peerswap-elementsd-swaps"
```